### ISSUE TITLE
The output of this model is similar when I feed the different style image

### ISSUE THREAD

**Issue by GesilaA** [CellEight/Pytorch-Adaptive-Instance-Normalization#1]
Thank you for you work! I clone this repository and download the pretrained weight from google drive.
Run the command `python style.py ${content_image} ${style_image}`, The output of the model are so similar. 
First col is the content image, second col is the style image, and the third col is the output image. Why this happen?
![merge](https://user-images.githubusercontent.com/19608156/128451237-b0c9417b-17d4-41bb-885d-1ab2140163af.jpg)
